Fabric.Authorization.Domain/Stores/Services/PermissionService.cs

```diff
@@ -118,17 +118,11 @@ public PermissionService(IPermissionStore permissionStore, RoleService roleServi
         {
             try
             {
-                var stored = await GetUserGranularPermissions(granularPermission.Target);
+                var stored = await GetUserGranularPermissions(granularPermission.Id);               
```

Comment:
Should this include the IdentityProvider in the query (dependent on question above about Id being the SubjectId only)?

Reply:
yes i noticed that too and am going to change it